distribution/docs/src/main/resources/_contents/_core-concepts/endpoints-intro-contents.adoc

```diff
@@ -1,7 +1,17 @@
 
-// to add
-Placeholder text.
-////
-TODO: write this section.
-See https://codice.atlassian.net/browse/DDF-2471
-////
+Endpoints expose the Catalog Framework to clients using protocols and formats that the clients understand.
+
+Endpoint interface formats/protocols can include a variety of formats, including (but not limited to):
```

Comment:
how about: 

`Endpoint interface formats encompass a variety of protocols, including (but not limited to):`